gno.land/pkg/gnoland/genesis.go

```diff
@@ -187,3 +188,29 @@ func LoadPackage(pkg gnomod.Pkg, creator bft.Address, fee std.Fee, deposit std.C
 
 	return tx, nil
 }
+
+func LoadEmbeddedStdlibs(deployer crypto.Address, fee std.Fee) []TxWithMetadata {
+	stdlibs := stdlibs.EmbeddedMemPackages()
+	stdlibsTxs := []TxWithMetadata{}
+
+	for _, memPkg := range stdlibs {
+		if memPkg.Path == "testing" {
```

Comment:
Major nitpick: can we make this a constant?

Reply:
1c2271868c81091bd51731df3c3749b2e2a1ded3